packages/harpparts/src/apparatus/constants/apparatus/WOOZLE_MINOR_HALF_VALVED_APPARATUS.ts

```diff
@@ -0,0 +1,32 @@
+import { BLOW, BLOWBEND1, DRAW, BEND1, BEND2, BEND3 } from '../interaction'
+import { ApparatusIds } from '../../types'
+import type { Apparatus, Interaction } from '../../types'
+import type { HarpFaceMatrix, HalfstepIndex } from '../../../types'
+
+// prettier-ignore
+const halfstepIndexMatrix: HarpFaceMatrix<HalfstepIndex> = [
+  //    1          2          3          4          5          6          7          8          9         10
+  [ -1       , 2        , 6        , 11       , 14       , 18       , 20       , 23       , 26       , 32        ], 
+  [ 0        , 3        , 7        , 12       , 15       , 19       , 21       , 24       , 27       , 33        ],
+  [ 2        , 7        , 10       , 14       , 17       , 21       , 22       , 26       , 31       , 36        ],
+  [ 1        , 6        , 9        , 13       , 16       , 20       , undefined, 25       , 30       , 35        ],
+  [ undefined, 5        , 8        , undefined, undefined, undefined, undefined, undefined, 29       , 34        ],
+  [ undefined, 4        , undefined, undefined, undefined, undefined, undefined, undefined, 28       , undefined ],
+] as const
+
+// prettier-ignore
+const interactionMatrix: HarpFaceMatrix<Interaction> = [
+  //    1          2          3          4          5          6          7          8          9         10
+  [ BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1, BLOWBEND1 ],
```

Comment:
It might be worth making a new bend type for valved bends. It's not clear what the benefit's are at this point, I think all the interaction types only exist in case we want to offer options to hide overblows etc which still isn't a feature I'm really considering so perhaps it's not important. If that's true then perhaps I can just delete the interaction matrix's.

Reply:
Done